packages/scaffold/src/partials/w3m-connecting-wc-mobile/index.ts

```diff
@@ -45,7 +45,11 @@ export class W3mConnectingWcMobile extends W3mConnectingWidget {
   private onBuffering() {
     if (document.visibilityState === 'visible' && !this.error) {
       this.buffering = true
```

Comment:
Since buffering was moved out into controler state, let's refactor here to where this.buffering subscribes to state changes instead of manually setting it.